.github/workflows/run_tests.yaml

```diff
@@ -12,11 +12,12 @@ on:
   push:
     branches:
       - master
+      - ac/nnunet_migration
```

Comment:
TODO: Revert this change once we're done testing this branch. (Note to self to avoid accidentally merging, hehe.)